package.json

```
@@ -39,25 +39,25 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~11.1.0",
-    "@angular/common": "~11.1.0",
-    "@angular/compiler": "~11.1.0",
-    "@angular/core": "~11.1.0",
-    "@angular/forms": "~11.1.0",
-    "@angular/platform-browser": "~11.1.0",
-    "@angular/platform-browser-dynamic": "~11.1.0",
-    "@angular/router": "~11.1.0",
+    "@angular/animations": "~11.2.11",
+    "@angular/common": "~11.2.11",
+    "@angular/compiler": "~11.2.11",
+    "@angular/core": "~11.2.11",
+    "@angular/forms": "~11.2.11",
+    "@angular/platform-browser": "~11.2.11",
+    "@angular/platform-browser-dynamic": "~11.2.11",
+    "@angular/router": "~11.2.11",
     "rxjs": "~6.6.3",
     "tslib": "^2.0.3",
     "zone.js": "~0.11.3"
   },
   "devDependencies": {
-    "@angular-builders/jest": "11.0.0",
-    "@angular-devkit/build-angular": "^0.1101.2",
+    "@angular-builders/jest": "11.2.0",
+    "@angular-devkit/build-angular": "^0.1102.10",
```

suggestion: Use version ranges that are not going to get us into trouble when Angular 12 is released.
```suggestion
    "@angular-builders/jest": "~11.2.0",
    "@angular-devkit/build-angular": "~0.1102.10",
```

---

Like what it had before?